org.quassel_irc.quasselclient.json

```diff
@@ -0,0 +1,36 @@
+{
+    "app-id": "org.quassel_irc.quasselclient",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "quasselclient",
+    "rename-icon": "quassel",
+    "rename-desktop-file": "quasselclient.desktop",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=x11",
+        "--share=network",
+        "--device=dri"
```

Comment:
QT apps usually add: "--env=QT_QPA_PLATFORM=flatpak", in order to get access to the KDE Portals.